src/plugins/vis_type_timeseries/public/application/editor_controller.js

```diff
@@ -17,10 +17,13 @@
  * under the License.
  */
 
+import './index.scss';
```

Comment:
Is this the right place to import all the styles for TSVB and not in the `plugin.ts` fle?

Reply:
Right now there is no benefit in importing it here in `editor_controller.js`,
since the plugin initialization will synchronously load these styles.
But this helps to decrease the plugin bundle if we'll switch loading of the `editor_controller.js` to a dynamic import in future. 

e.x. like we did it in the `dashboard` plugin:
styles are loaded in :
https://github.com/elastic/kibana/blob/master/src/plugins/dashboard/public/application/application.ts#L20

but this file is under a dynamic import, so it will be loaded only if the dashboard is rendered in browser:
https://github.com/elastic/kibana/blob/master/src/plugins/dashboard/public/plugin.tsx#L246

Reply:
But the styles included in this `index.scss` file aren't restricted to the editor application, right? There looks to be styles associated with just the render of the chart, which would need to be added to Dashboard, or any other place the TSVB chart types are displayed/embedded. So I definitely agree that importing at a lower component is a good idea, I think this is too low for **all** TSVB styles.

Reply:
Makes sense, moved into `plugin.ts`